src/components/Header/index.tsx

```diff
@@ -79,3 +59,46 @@ export function Header() {
         </header>
     );
 }
+
+export function PrimaryNav({ colorMode, toggleColorMode }: Props) {
+    return (
+        <div className="d-flex justify-content-between w-100">
+            <Navbar.Brand as={Link} href="/">
+                {SITE_NAME}
+            </Navbar.Brand>
+            <Navbar.Toggle aria-controls="main-navbar-nav" />
+            <Nav className="d-none d-lg-flex align-items-center">
+                <ThemeToggleButton
+                    colorMode={colorMode}
+                    toggleColorMode={toggleColorMode}
+                />
+            </Nav>
+        </div>
+    );
+}
+
+export function MobileNav({ colorMode, toggleColorMode }: Props) {
```

Comment:
Please move this to its own file in `src/components/Header/MobileNav.tsx`

Reply:
done